strr-base-web/app/composables/useStrrBasePermit.ts

```diff
@@ -50,6 +50,7 @@ export const useStrrBasePermit = <R extends ApiRegistrationResp, A extends ApiAp
       if (applications.length) {
         // Set active strata to the most recent application (ordered by api: newest to oldest)
         application.value = applications[0]
+        await updatePaymentDetails<A>(application.value?.header.applicationNumber as string)
```

Comment:
had to add this to not have 'payment due' when redirected to dashboard on platforms